lib/web_console/testing/erb_helper.rb

```diff
@@ -0,0 +1,24 @@
+require 'web_console/testing/fake_middleware'
+
+# This class is to pre-compile 'templates/*.erb'.
+class ERBHelper
```

Comment:
Oh, also, please put it inside of the `WebConsole::Testing` namespace. Its more predictable to have the `web_console/testing/erb_helper` class/module in this namespace. You can call it with the fully qualifided name wherever its referenced now. Same applies for the `FakeMiddleware`.